index.js

```diff
@@ -1,23 +1,20 @@
 'use strict';
-var childProcess = require('child_process');
 var arrify = require('arrify');
+var execa = require('execa');
+var Promise = require('pinkie-promise');
 
-module.exports = function (input, opts, cb) {
-	if (process.platform !== 'win32') {
-		throw new Error('Windows only');
-	}
+module.exports = function (input, opts) {
+	opts = opts || {};
+	input = arrify(input);
```

Comment:
Made it an array to circumvent XO complaining about `==`. Else, we could add an exception for it but I thought this was somewhat smarter.